TypeError Request User Does Not Exist

**System:**
- NodeJS 12.18.1

**Error:**
When performing a `yarn dev`

```bash
❯ yarn dev
yarn run v1.19.2
$ ts-node-dev src/server.ts
Using ts-node version 8.10.2, typescript version 3.9.6
Compilation error in /path/to/project/src/controllers/auth/logout.ts
[ERROR] 08:42:48 ⨯ Unable to compile TypeScript:
src/controllers/auth/logout.ts:20:54 - error TS2339: Property 'user' does not exist on type 'Request<ParamsDictionary, any, any, ParsedQs>'.
```

**Steps To Reproduce**

1. Run `yarn` to install dependencies
3. Run `yarn dev`

**Solution**

Modify `tsconfig.json` to add

```json
{
    "typeRoots": [
      "./src/types"
    ]
}
```

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -44,7 +44,7 @@
     // } /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */,
     // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
     "typeRoots": [
-      // "./types"
+      "./src/types"
     ] /* List of folders to include type definitions from. */,
     "types": [
       "./node_modules/@types/node",

